static/js/server_events_dispatch.js

```diff
@@ -627,6 +627,16 @@ export function dispatch_normal_event(event) {
                     message_list.narrowed.rerender();
                 }
             }
+            if (event.setting_name === "enter_sends") {
+                page_params.enter_sends = event.setting;
+                $("#enter_sends").prop("checked", page_params.enter_sends);
+                if (page_params.enter_sends) {
+                    $("#compose-send-button").fadeOut();
+                } else {
+                    $("#compose-send-button").fadeIn();
+                }
+                break;
+            }
```

Comment:
Maybe this should become a `compose.toggle_entry_sends_ui` type function?  We generally try to keep this sort of actual logic out of `server_events_dispatch.js`.